R/sgdnet-package.R

```diff
@@ -16,6 +16,7 @@
 
 #' @useDynLib sgdnet, .registration = TRUE
 #' @importFrom Rcpp sourceCpp
+#' @import methods
```

Comment:
Why are we importing the whole `methods' package? AFAIR, we have no S4 anywhere in this code (except possibly sparse matrix support, but we shouldn't need `methods` imported for that.)

Reply:
I have removed the methods' package. Some earlier travis ci wanted it, but now it is fine.